config/resources/configs/frequency-rococo.json

```diff
@@ -0,0 +1,35 @@
+{
+  "maxGraphPageSizeBytes": 2048,
```

Comment:
1024

Reply:
right; I put in placeholders & then forgot to check in the frequency repo 🤦🏻